### PR TITLE
feat: added letterboxd social button

### PR DESCRIPTION
**Pull request by PeculiarGoat**
[ADO Card](https://dev.azure.com/S72/SHIFT72/_workitems/edit/4189)

Screenshot_2021-09-13_22-25-38

This can't be merged until the Kibble part is released.

If anyone has a better idea on how to do that icon please tell me!

### CODE REVIEW

File: site/templates/common/social-media-buttons.jet [hunk 1]
```diff
@@ -13,6 +13,15 @@
 			<a class="btn btn-social btn-social-linkedin" aria-label="{{i18n("wcag_aria_label_linkedin")}}" href="https://www.linkedin.com/shareArticle?mini=true&url={{site.SiteConfig.SiteURL + path | url}}&title={{title|url}}" target="_blank">
 				<i class="fa fa-linkedin" title="Share on LinkedIn" aria-hidden="true"></i>
 			</a>
+
+			{{ if letterboxdID }}
```
Comment (stajs) [inline site/templates/common/social-media-buttons.jet:17]: Nit: remove spaces.
```suggestion
			{{if letterboxdID}}
```

**Review by stajs — APPROVED** [review]
LGTM, side-question though: what are we going to do about other languages when introducing a new translation key?

File: site/templates/common/social-media-buttons.jet [hunk 2]
```diff
@@ -13,6 +13,15 @@
...
+				<a class="btn btn-social btn-social-letterboxd" aria-label="{{i18n("wcag_aria_label_letterboxd")}}" href="https://boxd.it/{{ letterboxdID }}" target="_blank">
+					<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="28px" height="28px" viewBox="0 0 560 560" version="1.1">
+						<title>{{i18n("wcag_aria_label_letterboxd")}}</title>
+						<path d="M185.422,140 L185.422,420 L374.578,420 L374.578,350.101 L258.146,350.101 L258.146,140 L185.422,140 Z"/>
+					</svg>
+				</a>
+			{{ end }}
```
Comment (smitsanghvi21) [inline site/templates/common/social-media-buttons.jet:24]: `{{end}}`

**Review by smitsanghvi21 — APPROVED** [review]

**Comment by PeculiarGoat** [comment]
@stajs Yeah I've been thinking about that and haven't got any solid ideas.  Maybe we would treat en_AU as the master list of keys and have some service or process tell us when the other languages are missing something?  Maybe we would want to force an uber user to input values for every language at once when they add a new key?  Maybe an English placeholder could be added to every language when a new key is added?

**Comment by stajs** [comment]
@PeculiarGoat Can you put those thoughts in to an ADO card? I think we need to have a way of validating what keys are required.

**Comment by PeculiarGoat** [comment]
@stajs https://dev.azure.com/S72/SHIFT72/_workitems/edit/5526

**Review by l0ud0gg — APPROVED** [review]
looks good after a quick read

**Review by imran-salim — APPROVED** [review]

**Pull request merged**